### PR TITLE
[Docs] Update Project pages

### PR DESCRIPTION
**Pull request by yaronha**

### CODE REVIEW

**Comment by george0st** [comment]
@yaronha only note: I am focusing on topic archive/central archive as the solution of long-term data keeping (10, 30 years, etc.), ... and from this reason maybe the term "Source respository" insteand of "Source archive" can be better.

**Review by gilad-shaham — APPROVED** [review]
Looks excellent, the comments I have are minor or outside the scope

File: docs/projects/create-project.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+```
+my-project           # Parent directory of the project (context)
+├── data             # Project data for local tests or outputs (not tracked by version control)
+├── docs             # Project documentation
+├── src              # Project source code (functions, libs, workflows)
+├── tests            # Unit tests (pytest) for the different functions
+├── project.yaml     # MLRun project spec file
+├── README.md        # Project README
+└── requirements.txt # Default Python requirements file (can have function specific requirements as well)
```
[the remark this answers is not in the document]
Comment (nschenone) [reply docs/projects/create-project.md:27]: We should make a [cookie cutter template](https://www.cookiecutter.io/templates)

File: docs/projects/load-project.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+You can edit or add project elements like functions, workflows, artifacts, etc. (See  [**create and use projects**](./create-project.html).)
+Once you make changes use GIT or MLRun commands to push those changes to the archive (See [**save into git or an archive**](./create-project.html#push).)
+
+<a id='load-cli'></a>
+## Load projects using the CLI
+
+Loading a project from `git` into `./` :
+
+```
+mlrun project -n myproj -u "git://github.com/mlrun/project-demo.git" .
```
Comment (nschenone) [inline docs/projects/load-project.md:53]: We should use the expanded flags for the CLI commands. Makes it a little more clear what exactly is happening (for learning purposes)

**Pull request merged**